cmd/tetragon/main.go

```diff
@@ -629,6 +615,10 @@ func execute() error {
 	// disable.
 	flags.Bool(keyReleasePinnedBPF, true, "Release all pinned BPF programs and maps in Tetragon BPF directory. Enabled by default. Set to false to disable")
 
+	// Provide option to enable policy filtering. Because the code is new,
+	// this is set to false by default.
+	flags.Bool(keyEnablePolicyFilter, false, "Enable policy (beta) filter code")
```

Comment:
Can we just not expose this? mark it hidden or at least add words here to say this will be turned on by default in future?
Users may start to rely on it being false, then complain later...

Reply:
No I'd like to expose it so that people can try this out.